redisinsight/ui/src/components/virtual-tree/interfaces.ts

```diff
@@ -1,7 +1,7 @@
 import { FixedSizeNodeData } from 'react-vtree'
 import { IKeyPropTypes } from 'uiSrc/constants/prop-types/keys'
 
-export const TREE_LEAF_FIELD = 'keys:keys'
+export const getTreeLeafField = (delimiter = '') => `keys${delimiter}keys`
```

Comment:
could you please move it from interfaces file?

Reply:
moved